scripts/dependencies/wsl.ts

```diff
@@ -54,70 +54,6 @@ export class HostSwitch implements Dependency, GitHubDependency {
   }
 }
 
-export class HostResolverPeer implements Dependency, GitHubDependency {
```

Comment:
Do we need to remove it from the dependencies list?
https://github.com/rancher-sandbox/rancher-desktop/blob/ce827861bc2f441f881c3f5dba7facf36f02ffcb/pkg/rancher-desktop/assets/dependencies.yaml#L18

Also the corresponding type:
https://github.com/rancher-sandbox/rancher-desktop/blob/ce827861bc2f441f881c3f5dba7facf36f02ffcb/scripts/lib/dependencies.ts#L51